docs/release-notes/7278-remove-log-option

```diff
@@ -0,0 +1,7 @@
+:orphan:
```

Comment:
this file should have `.rst` extension and `make fmt` applied.

Reply:
✔️ thanks